manager.go

```diff
@@ -287,7 +287,8 @@ func (m *Manager) prunePeers() {
 			count++
 			continue
 		}
-		if now.Sub(node.LastSuccess) > pruneExpireTimeout {
+		if !node.LastAttempt.IsZero() &&
```

Comment:
Without screening out unattempted nodes, il-timed firing of pruneAddressTicker would prune a new peer that is presently being tested.

The deferred timestamping of the attempt is necessary pruning peers during the actual test.